app/assets/javascripts/discourse/app/components/topic-timeline/container.hbs

```diff
@@ -137,5 +138,6 @@
         <TopicAdminMenuButton @topic={{@model}} @addKeyboardTargetClass={{true}} @openUpwards={{true}} />
       {{/if}}
     {{/if}}
+    <PluginOutlet @name="timeline-footer-controls-after" @tagName="span" @connectorTagName="span" @args={{hash model=@model}}/>
```

Comment:
Let's drop the `tagName` and `connectorTagName`. The default now is for tagless outlets, which give the most control to plugins/themes. (and are the most glimmer-like)